packages/image_picker/example/android/app/src/test/java/io/flutter/plugins/imagepicker/ImagePickerPluginTest.java

```diff
@@ -109,6 +109,15 @@ public void onResiter_WhenAcitivityIsNull_ShouldNotCrash() {
         "No exception thrown when ImagePickerPlugin.registerWith ran with activity = null", true);
   }
 
+  @Test
+  public void onConstructor_WhenContextTypeIsActivity_ShouldNotCrash() {
+    // When you use Module to dependency, the type of Context will be Application.but when you use AAR to dependency on this plugin, the type of Context will be Activity.
```

Comment:
Nits: The test case name is pretty self explanatory. We can remove this comments to reduce some noise.